fiducialMarker/fiducialMarker.py

```diff
@@ -1,9 +1,10 @@
 from cv2 import aruco
 
 class FiducialMarker:
+    __dictionary = aruco.getPredefinedDictionary(aruco.DICT_4X4_50)

```

Comment:
So 4 bits by 4 bits = 16 bit squares?
16 bit squares either ON or OFF. 2^16 = 65536 unique marker possibilities.
Marker corpus of 50 id's possible (MAX)?
coolio.